spec/steps/categories/categories_maintaining_steps.rb

```diff
@@ -47,6 +47,7 @@ module CategoryMaintainingSteps
 
   step 'I should not see category on categories list page' do
     category_name = category.name
+    sleep 2
```

Comment:
We have to remove it. This is antipattern. If you create category via UI in action, then you have to check there with matcher, not in Verification and not with sleep

Reply:
replaced with 'is_expected.to have_no' matcher